src/server.ts

```diff
@@ -7,6 +9,8 @@ server.get('/', async (req, res) => {
     res.send('Hello World!');
 });
 
+server.use('/', ensureAuthenticated(), emailRouter);
```

Comment:
Seems like auth is used as middleware currently for sending emails, in the future we will need dedicated endpoint for auth and account info to support login on the frontend

Reply:
Just added an endpoint for auth. Might need further work on how we handle access tokens and storing it for other APIs to use. I'm still using an ensureAuthenticated() helper for now.